### DISCUSSION TITLE
Unable to Add New Build Output in InvenTree Demo

### DISCUSSION THREAD

**Discussion by palagap** [inventree/InvenTree#7314] [Q&A]
Hi there,

While using the InvenTree demo version 0.16.0 dev, I am having an issue creating a new build output. In the demo, while logged in as admin, I created a new part (Test Assembly 1), and added two parts to the bill of materials. I then created a build order (BO0029) for Test Assembly 1. However, when trying to add a “New Build Output” to the build order, I encountered the error: "BuildOutputCreateSerializer" object has no attribute 'Meta'.    

Screenshots of the error are below:

Error as seen on the Build Order page:

![image](https://github.com/inventree/InvenTree/assets/40808444/dea99129-649f-47ad-9bf6-012185ff9ddf)

Error as seen on the Admin page:

![image](https://github.com/inventree/InvenTree/assets/40808444/5eec1ee1-6b4e-4a45-ac27-8f536efe4d4f)

Are there any suggestions on how to proceed? Thanks.

**Comment by SchrodingersGat** [comment]
Hi @palagap looks like this is a bug, I'll have a look into this.

Edit: See https://github.com/inventree/InvenTree/pull/7315

**Reply by palagap** [reply]
Great, thank you.
